meta-edgefarm-core/recipes-containers/docker-prune/files/docker-prunea-at-boot.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Clean Up Docker Containers
```

Comment:
When is the service executed? It should run after docker.service and before edgecore.service.

Reply:
Could not test it, but looks good for me.